docs/080-breaking-changes.rst

```diff
@@ -17,3 +17,10 @@ and it does something else afterwards.
 * Explicit conversions from negative literals and literals larger than ``type(uint160).max`` to ``address`` are now disallowed.
 * Exponentiation is right associative, i.e., the expression ``a**b**c`` is parsed as ``a**(b**c)``.
   Before 0.8.0, it was parsed as ``(a**b)**c``.
+
+Syntactic Only Changes
```

Comment:
"Syntactic Changes" or "Syntax Only Changes" I think are more correct.

Reply:
We have to re-group them somehow. It's also not really a "syntax only change"